### ISSUE TITLE
Arm64 support?

### ISSUE THREAD

**Issue by jambots** [ChoiZ/NowPlaying#6]
On Phonegap Build 3.7.0 'Error - A plugin in your app does not support the compulsory arm64 (64-bit support) architecture.'

**Comment by ChoiZ** [comment]
Are you using only NowPlaying plugin? Can you disable all plugins and try again?

**Comment by jambots** [comment]
Yes, it builds fine with plugins removed from config.xml Please find
attached screenshots of error when <gap:plugin
name="com.rd11.remote-controls" version="1.0.0" /> added.


On Wed, Mar 18, 2015 at 3:09 AM, François LASSERRE <notifications@github.com
> wrote:

> Are you using only NowPlaying plugin? Can you disable all plugins and try
> again?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ChoiZ/NowPlaying/issues/6#issuecomment-82811820>.
>



-- 
-Terry Schubring
http://jambots.com
651-767-2386

The crowning fortune of a man is to be born to some pursuit which finds him
employment and happiness, whether it be to make baskets, or broadswords, or
canals, or statues, or songs.
-- Ralph Waldo Emerson

**Comment by jambots** [comment]
When was the plugin compiled last? The 64-bit requirement for new apps added to the store began February 1st, 2015. A specific test would be to create a new app and use the App Loader to upload an IPA with the plugin included. If that proves the plugin is 64-bit then the error must be caused by phonegap build.

**Comment by ChoiZ** [comment]
My last test was done more than a year ago (January 2014). I can't fix the 64-bit problem now, but if you have time you can make a pull request ;).
